test/repository_test.rb

```diff
@@ -98,6 +105,10 @@
             UserRepository.create(user1),
             UserRepository.create(user2)
           ]
+
+          @created_at        = DateTime.new(2015,4,1)
+          @unpersisted_phone = Phone.new(name: 'Iphone 5S', publisher: 'Apple')
+          @persisted_phone   = Phone.new(name: 'Galaxy s5', publisher: 'Samsung', created_at: @created_at)
```

Comment:
This is not a persisted entity because it does not have id. Please assign id, just random number